Cargo.toml

```diff
@@ -22,49 +22,97 @@ path = "src/bindgen/bin.rs"
 required-features = ["build-binary"]
 
 [dependencies]
+
 log = "0.4.20"
-pretty_env_logger = "0.5.0"
-derive_more = { version = "1.0.0-beta.6", features = [
+# log = { git = "https://github.com/rust-lang/log/", rev = "4708f1484c7e6b8d4418b571c05e613b18e57673" }
```

Comment:
commented? shouldn't it replace the `log` above or maybe you meant to comment out the one above :)?

Reply:
Does not work.  Will need help from builders team